flink-table/flink-table-planner/src/main/java/org/apache/flink/table/planner/expressions/converter/ExpressionConverter.java

```diff
@@ -64,8 +64,8 @@
 import java.util.Optional;
 import java.util.stream.Collectors;
 
+import static org.apache.flink.table.planner.utils.TimestampStringUtils.fromLocalDateTime;
```

Comment:
maybe hotfix commit, since nothing else has changed in this file.

Reply:
Not sure it brings much value to split it in a separate commit, but it should build fine separetely. I guess it's up to the committer who's going to merge this PR :)

Reply:
yeah, I know, it's very nit, and only to provide a clean git history.